packages/pwa/app/translations/README.md

```diff
@@ -75,10 +77,12 @@ During your development, you extract and compile the messages and pass the gener
 
 We’ve provided a few npm scripts to help you:
 
--   `npm run extract-messages` to extract the default messages
+-   `npm run extract-default-translations` to extract the default messages
     -   so you don’t have to create the JSON file yourself by hand
--   `npm run compile-messages` to compile all locales into Abstract Syntax Tree (AST) format
+-   `npm run compile-translations` to compile all locales into Abstract Syntax Tree (AST) format
     -   `react-intl` can use this AST format to bypass the parsing step and thus, save a bit of time.
+-   `npm run build-translations` to run both of the above
+    -   run this whenever you add a new formatted message to your components to update the default locale's translations
```

Comment:
🤔 do we want to suggest to simply have this `build-translation` as the 1 command to remember?